backend/src/resolvers/comments.spec.js

```diff
@@ -0,0 +1,62 @@
+import Factory from '../seed/factories'
+import { GraphQLClient } from 'graphql-request'
+import { host, login } from '../jest/helpers'
+
+const factory = Factory()
+let client
+let variables
+
+beforeEach(async () => {
+  await factory.create('User', {
+    email: 'test@example.org',
+    password: '1234'
+  })
+})
+
+afterEach(async () => {
+  await factory.cleanDatabase()
+})
+
+describe('CreateComment', () => {
+  const mutation = `
+    mutation($id: ID!, $content: String!) {
+      CreateComment(id: $id, content: $content) {
```

Comment:
Adding the post here might get tricky. The author-to-post relationship gets created via request parameters, while comment-to-post needs to be created through additional input params.